packages/rocket-batch-file-process-aws-infrastructure/src/file-to-line-event-lambda/index.ts

```diff
@@ -0,0 +1,90 @@
+import { S3Event } from 'aws-lambda'
+import { DynamoDB, S3 } from 'aws-sdk'
+
+const CSV_FILE = 'csv'
+const JSONL_FILE = 'jsonl'
+
+const s3 = new S3()
+const ddb = new DynamoDB.DocumentClient()
+const { v4: uuidv4 } = require('uuid')
+
+const ENTITY_ID_NAME = process.env.ENTITY_ID!
```

Comment:
If we want to use environment variables in rockets, we should scope them somehow too to avoid inter-rocket collisions. Ideally, all parameters should be provided via config.

Reply:
This env variable is used only in one lambda of the rocket and its scope is only that lambda function. 
Not sure what you mean by `Ideally, all parameters should be provided via config.`